src/sdk/clients/createBicoPaymasterClient.test.ts

```diff
@@ -74,14 +82,14 @@ describe.runIf(paymasterTruthy())("bico.paymaster", async () => {
 
     nexusAccountAddress = await nexusAccount.getCounterFactualAddress()
 
-    nexusClient = await createNexusClient({
+    nexusClient = (await createNexusClient({
       signer: account,
       chain,
       transport: http(),
       bundlerTransport: http(bundlerUrl),
       paymaster,
       ...testParams
-    })
+    })) as NexusClient
```

Comment:
Do you need to cast this? Just curious because if you do we should fix 🤔

Reply:
No, fixed